mlem/contrib/kubernetes/context.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import os
+from typing import ClassVar, Optional
+
+from pydantic import BaseModel
+
+from mlem.utils.templates import TemplateModel
+
+logger = logging.getLogger(__name__)
+
+
+class K8sYamlBuildArgs(BaseModel):
+    name: Optional[str] = "mlem-app"
+    image: Optional[str] = "mlem-app"
+    image_pull_policy: Optional[str] = "Always"
+    port: Optional[int] = 8080
+    service_type: Optional["str"] = "NodePort"
+
+
+class K8sYamlGenerator(K8sYamlBuildArgs, TemplateModel):
```

Comment:
You don't need separate Args class, since you only use it once. Docker args are used multiple times if I remember correctly